internal/storage/storage_handle.go

```diff
@@ -82,6 +83,7 @@ func NewStorageHandle(ctx context.Context, clientConfig StorageClientConfig) (sh
 			Base:   transport,
 			Source: clientConfig.TokenSrc,
 		},
+		Timeout: clientConfig.HttpClientTimeout,
```

Comment:
do you want to add a test case which passes timeout to ensure nothing breaks.

https://github.com/GoogleCloudPlatform/gcsfuse/pull/934/files#diff-2d5c5e3691214ba273ac9050fc100dcfff9a5981d5844f74e653718f05ae10dd

Reply:
Sure, added